flake.nix

```diff
@@ -45,15 +42,12 @@
         #     riscv-arch = "rv32imc";
         #   };
 
-        pythonEnv = pkgs.python3.withPackages (
```

Comment:
To me it makes more sense to assemble python environments on a per-project basis, pulling in patched versions of dependencies from lowrisc-nix when needed. I don't really see the point of keeping the env itself in the common repo?

Reply:
I have converted the env to use poetry